src/components/Form/Form.js

```diff
@@ -0,0 +1,24 @@
+import Button from "./Button";
+import { handleSubmit } from "./Button";
+import "../Main/Main.css";
+import "./Button.css";
+
+export default function Form() {
+  return (
+    <>
+      <section className="heading">
+        <h2>NEW ENTRY - TODAY, FEB 28, 2028</h2>
+      </section>
+
+      <form className="form">
+        <label for="motto">Motto</label>
+        <textarea id="motto" rows="1"></textarea>
+        <label for="notes">Notes</label>
```

Comment:
Die Labels werfen in der Konsole einen Fehler, da du das "for"-Attribut verwendest. Bei JSX müsste es aber das "htmlFor" sein